addons/web/controllers/kanban.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import http, _
+from odoo.http import request
+from odoo.exceptions import UserError
+from odoo.osv import expression
+
+CATEGORY_SUPPORTED_TYPES = ['many2one']
+FILTER_SUPPORTED_TYPES = ['many2one', 'many2many', 'selection']
+
+
+class KanbanSearchPanel(http.Controller):
+
+    @http.route('/web/kanban/get_search_panel_category', type='json', auth="user")
+    def get_search_panel_category(self, model, field_name):
```

Comment:
add docstring

Reply:
changed into model method; comment is no longer relevant